### PR TITLE
 Edits on edit expense page

### PR DESCRIPTION
**Pull request by El-amiir**

### CODE REVIEW

File: public/expenses/edit.php [hunk 1]
```diff
@@ -20,93 +20,106 @@
...
+    let inputs = document.querySelectorAll('.expense__form--content-input');
+    console.log(inputs);
+    inputs.forEach(input=>{
+        input.addEventListener("mouseover", function(e){
+          e.target.parentElement.parentElement.firstElementChild.style.letterSpacing = ".8rem";
+        })
+        input.addEventListener("mouseout", function(e){
+          e.target.parentElement.parentElement.firstElementChild.style.letterSpacing = ".3rem";
+        })
+    })
```
Comment (mohamed-abdul-fattah) [inline public/expenses/edit.php:121]: This code fragment is duplicate with the fragment in the add  page and could be extracted to a Js script file and included in both pages.

**Review by mohamed-abdul-fattah — COMMENTED** [review]
Good job 👍🏼. Kindly, check my minor comment below.

File: public/expenses/edit.php (hunk 1 above)
Comment (El-amiir) [reply public/expenses/edit.php:121]: Done in next pull request (issue-16-Delete-Modal)

**Review by mohamed-abdul-fattah — APPROVED** [review]
LGTM

**Pull request merged**